Woof/Woof/Model/Preferences/Preferences.swift

```diff
@@ -0,0 +1,8 @@
+import Foundation
+
+/// Represents user preferences.
+struct Preferences: Codable {
```

Comment:
Do we need a conformance to `Codable` at the moment?

Reply:
We store Preferences in KeyValueStorage as Data, that's why we need to conform to the Codable Protocol.